CatProducer/python/catTools_cff.py

```diff
@@ -91,27 +68,9 @@ def catTool(process, runOnMC=True, useMiniAOD=True):
 #        )
         #process.patJetsUpdated.userData.userFloats.src +=['pileupJetIdUpdated:fullDiscriminant']
 
-#        process.catJetsPuppi.src = cms.InputTag("patJetsPuppiUpdated")
-#        process.catJetsPuppi.setGenParticle = cms.bool(False)
-        ## #######################################################################
-        ## qg-likelihood
-        # check https://twiki.cern.ch/twiki/bin/viewauth/CMS/QGDataBaseVersion
-#        from CATTools.CatProducer.patTools.jetQGLikelihood_cff import enableQGLikelihood
-#        process = enableQGLikelihood(process, qgDatabaseVersion="v2b", runOnMC=runOnMC, useMiniAOD=useMiniAOD)
-
-        ## DeepFlavour
-        from CATTools.CatProducer.patTools.jetDeepFlavour_cff import enableDeepFlavour
-        process = enableDeepFlavour(process)
-
-        ## #######################################################################
-        ## # MET corrections from https://twiki.cern.ch/twiki/bin/view/CMS/MissingETUncertaintyPrescription
-#       from PhysicsTools.PatUtils.tools.runMETCorrectionsAndUncertainties import runMetCorAndUncFromMiniAOD
-#       runMetCorAndUncFromMiniAOD(process, isData= not runOnMC, electronColl=cms.InputTag('calibratedPatElectrons'))
-#       process.catMETs.src = cms.InputTag("slimmedMETs","","CAT")
-
-        #from CATTools.CatProducer.patTools.metMuonRecoMitigation2016_cff import enableMETMuonRecoMitigation2016
-        #process = enableMETMuonRecoMitigation2016(process, runOnMC) ## MET input object is overridden in the modifier function
 
+    ## #######################################################################
+    ## # MET corrections from https://twiki.cern.ch/twiki/bin/view/CMS/MissingETUncertaintyPrescription
     if useMiniAOD:
```

Comment:
(can be ignored) I think we can remove these "useMiniAOD" flags at some point because we _always_ use MiniAOD.